p2panda-net/src/sync/manager.rs

```diff
@@ -248,12 +258,19 @@ impl SyncManager {
         let sync_protocol = self.sync_protocol.clone();
         let engine_actor_tx = self.engine_actor_tx.clone();
 
+        let topic = self
+            .topic_map
+            .get(&topic)
+            // @TODO: I'm not sure if this case can ever occur, if it can, it would be best to
```

Comment:
I believe the returned value will always be `Some`; `connect_and_sync()` will only be called if a sync attempt has been scheduled (and that requires a `ToSyncManager` message with topic).